src/sentry/static/sentry/app/views/projectTags.jsx

```diff
@@ -39,13 +40,12 @@ export default class ProjectTags extends AsyncView {
   renderLink(key, canDelete, idx) {
     return (
       <LinkWithConfirmation
-        className={'btn btn-sm btn-default'}
         title={'Remove tag?'}
         message={'Are you sure you want to remove this tag?'}
         onConfirm={() => this.onDelete(key, idx)}
         disabled={!canDelete}
       >
-        <span className="icon icon-trash" />
+        <Button size="xsmall" icon="icon-trash" className="ref-delete" />
```

Comment:
use `data-test-id` instead of a `className`

Reply:
Thanks, I had seen refs used in other places

Reply:
🙏